frontend/src/pages/SketchListPage.tsx

```diff
@@ -1,14 +1,17 @@
 import { useEffect } from "react";
+import { useAtom } from "jotai";
 
+import { Modal } from "~/shared/components/Modal";
 import { useAppDispatch, useAppSelector } from "~/shared/hooks";
 import { APIService } from "~/shared/services";
 import { SketchList } from "~/features/sketches/SketchList";
 import { InfiniteCanvas } from "~/shared/components/InfiniteCanvas";
+import { SelectedSketchUrl } from "~/shared/store/Sketch";
 
 export function SketchListPage() {
   const { sketches } = useAppSelector((state) => state.sketches);
   const dispatch = useAppDispatch();
-
+  const [selectedSketchUrl, setSelectedSketchUrl] = useAtom(SelectedSketchUrl);
   useEffect(() => {
```

Comment:
[nits]
まとまりごとに1行空けた方が構造的に読みやすいと思う！
```suggestion
  const [selectedSketchUrl, setSelectedSketchUrl] = useAtom(SelectedSketchUrl);

  useEffect(() => {
```